components/PageHeader.vue

```diff
@@ -10,28 +10,37 @@
       <span>{{ $t('最終更新') }} </span>
       <time :datetime="formattedDate">{{ date }}</time>
     </div>
+    <div class="annotation">
```

Comment:
日本語の場合に非表示にする方法
```suggestion
    <div class="annotation" v-if="$i18n.locale !== 'ja'">
```

Reply:
日本語とやさしい日本語で非表示にする方法
```suggestion
    <div v-if="!['ja', 'ja-basic'].includes($i18n.locale)" class="annotation">
```